### PR TITLE
github: apiclarity to openclarity org move

### PR DESCRIPTION
**Pull request by FrimIdan**

### CODE REVIEW

**Review by klyr — APPROVED** [review]

File: charts/apiclarity/Chart.yaml [hunk 1]
```diff
@@ -4,9 +4,9 @@ appVersion: latest
 description: Charts for APIClarity deployment. Reconstruct OpenAPI Specifications from real-time workload traffic seamlessly.
```
Comment (jadiaconu) [inline charts/apiclarity/Chart.yaml:4]: Shall the name APIClarity also change ?

File: charts/apiclarity/Chart.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ appVersion: latest
...
 version: latest
 home: https://apiclarity.io/
```
Comment (jadiaconu) [inline charts/apiclarity/Chart.yaml:6]: Same question for the domain ?

**Review by akpsgit — APPROVED** [review]

File: charts/apiclarity/Chart.yaml (hunk 1 above)
Comment (jadiaconu) [reply charts/apiclarity/Chart.yaml:4]: Ok, so only org changes, ignore this

File: charts/apiclarity/Chart.yaml (hunk 2 above)
Comment (jadiaconu) [reply charts/apiclarity/Chart.yaml:6]: Ok, so only org changes, ignore this

**Review by jadiaconu — APPROVED** [review]

**Pull request merged**